docs/StardustDocs/topics/convert.md

```diff
@@ -76,3 +80,21 @@ dataFrameOf("direction")("NORTH", "WEST")
 
 <dataFrame src="org.jetbrains.kotlinx.dataframe.samples.api.Modify.convertToEnum.html"/>
 <!---END-->
+
+And finally, [Value classes](https://kotlinlang.org/docs/inline-classes.html) can be created
+and unpacked using `convert`:
```

Comment:
Just a small note, what do you mean "unpacked"? One can convert IntClass to let's say Double?

Reply:
I'll rewrite to clarify it a bit. Yes, you can convert IntClass to Double and back :). Didn't know this before